bundler/lib/bundler.rb

```diff
@@ -167,6 +171,22 @@ def setup(*groups)
       end
     end
 
+    def install
+      Bundler.definition.specs
+    rescue GemNotFound, GitError
+      Bundler.ui.info "Automatically installing missing gems."
```

Comment:
I was thinking through if this would ever show up when it wasn't "automatic", and I don't think so. `bundle install` calls `Bundler::Installer.install` the same as this method does, so this should be the only time it happens automatically.

Reply:
I think you're right!